guides/v2.3/graphql/reference/store-config.md

```diff
@@ -69,7 +67,18 @@ Attribute |  Data Type | Description
 `no_route` | String | Contains the URL of the default page that you want to appear when if a 404 “Page not Found” error occurs
 `show_cms_breadcrumbs` | Int | Determines if a breadcrumb trail appears on all CMS pages in the catalog. Options: `0` (No) or `1` (Yes)
 
+### Supported website attributes
```

Comment:
Delete this section. I recently merged a PR that added this info to the `master` branch.

Reply:
**done**